msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+
+
+> **Important**: Changing any of these settings doesn't affect data access for apps that were already granted consent. 
+
+## Configure Teams App to support RSC (Developer)
+
+The basic steps required to configure a service and get a token from the Microsoft identity platform endpoint that your service can use to call Microsoft Graph under its own identity are:
+
+1. Register your application in Azure AD.
+2. Check permissions configured in the 'API Permissions' section.
+3. Update your Teams app manifest to link to your AAD app ID.
+4. Get an access token.
+5. Make a Graph call.
+
+
+#### 1. Register a new application in Azure AD
+
+[Register](/graph/auth-register-app-v2) a new application in the [Azure Active Directory- App Registration portal](https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) to get an AAD app ID. If your AAD app supports [single sign-on (SSO) for tabs](/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso), then use the same AAD app ID for configuring RSC.
+>[!Note]
+>It is not a good idea to have multiple Teams apps link to the same AAD app ID. Team AAD app ID must be unique – if anyone tries to install and link multiple Teams app to the same AAD app ID, the installation will fail.
+
+#### 2. Check permissions configured in the 'API Permissions' section
+RSC permissions are only configured in the app manifest file. However, we need to check permissions in the 'API permissions' section in the app registration portal. Navigate to the 'API Permissions' section to check for any RSC and non-RSC permissions. If your app is newly registered, it may not have any additional permissions configured. Hence you can proceed further to step 3 to configure RSC permissions in the app manifest file. On the other side, if your app is an existing app, it may have additional permissions configured. In this case, ensure there are no RSC permissions configured here as we will configure them in the app manifest file as part of step 3. You can retain the non-RSC permissions like part of any normal app registration process.
+
+E.g. Graph permissions are not always about Teams data.
+For example, Your AAD app may need Mail.Read permission, but this is not an RSC scenario because it is not about team data, and team owners cannot give consent to other's mailboxes. APIs for Files, SharePoint, OneNote, Planner, and Calendar within a team do not support RSC yet.
+
+#### 3. Update your Teams app manifest to link to your AAD app ID
+Add a [webApplicationInfo](/microsoftteams/platform/resources/schema/manifest-schema#webapplicationinfo) section to the manifest that includes two properties - `id` is the AAD app ID, and `applicationPermissions` are the RSC permissions (scopes) your app requires.
+
+> **Note**: If your app needs non-RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
```

Comment:
Mention that RSC permissions are the ones that end in ".Group"

Reply:
Done